flink/v1.15/flink/src/test/java/org/apache/iceberg/flink/source/TestIcebergSourceContinuous.java

```diff
@@ -403,7 +406,7 @@ public static List<Row> waitForResult(CloseableIterator<Row> iter, int limit) {
 
   public static void waitUntilJobIsRunning(ClusterClient<?> client) throws Exception {
```

Comment:
remove this method and instead of this it should be something like
`Awaitility.await().pollDelay(10, TimeUnit.MILLISECONDS).until(() -> !getRunningJobs(MINI_CLUSTER_RESOURCE.getClusterClient()).isEmpty());`